.github/workflows/automationTemplate.yaml

```diff
@@ -75,7 +79,7 @@ jobs:
         with:
           branch: automation/add-new-${{ inputs.platformName }}-versions-${{ env.RELEASE_DATE }}
           title: New ${{ env.PLATFORM_NAME_CAMEL_CASE }} Sdks ${{ env.RELEASE_DATE }}
-          token: ${{ secrets.GH_TOKEN }}
+          token: ${{ secrets.PAT_TEMP }}
```

Comment:
Could you please confirm we have a work item to not forget to change that soon? Also, my understanding is that we need this today so that we unblock the pipelines. Is this correct?

Reply:
Yes, this unblocks us from the issue. I have two workitems created and assigned to me for now: `1894797` (Allowing GH token to create PR) and `1894896` (Refresh PAT to make sure the token is still valid). The token refresh should be just performing logging in from my side every 30 days according to Microsoft security policy in their org account.

Reply:
OK thanks!